### ISSUE TITLE
Bintray is shutting down

### ISSUE THREAD

**Issue by noorul** [spark-jobserver/spark-jobserver#1358]
Got this e-mail today

<img width="584" alt="Screenshot 2021-04-01 at 2 42 07 PM" src="https://user-images.githubusercontent.com/155376/113271991-c3d51b00-92f8-11eb-9efc-5d6c638b5a6f.png">

**Comment by velvia** [comment]
Yeah I forwarded this too, we’ll have to change to JFrog

**Comment by murraytodd** [comment]
Um, are build systems going to start failing the day after tomorrow?

**Comment by valan4ik** [comment]
@murraytodd JFrog updated two days ago [their post about JCenter being shut down](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/). It looks like Bintray should continue to work and there is a bit more time for the migration:
>UPDATE 4/27/2021: We listened to the community and will keep JCenter as a read-only repository indefinitely. Our customers and the community can continue to rely on JCenter as a reliable mirror for Java packages.

**Comment by murraytodd** [comment]
Okay, thanks for the note. I didn't want our Jenkins system to start failing on our builds over the weekend.

**Comment by vimal3271** [comment]
http://dl.bintray.com/ is forbidden. For fresh builds the download of spark job server jar is failing.

**Comment by velvia** [comment]
We are working on a migration to JFrog at the moment.

**Comment by valan4ik** [comment]
I just created a new release `0.11.1` and this release should be available after adding an additional resolver in `build.sbt`:

```
resolvers += "Artifactory" at "https://sparkjobserver.jfrog.io/artifactory/jobserver/"
```

**Comment by valan4ik** [comment]
Hi everyone,
Version `0.10.2` is now also available on JFrog, and README documentation is updated: https://github.com/spark-jobserver/spark-jobserver/blob/master/README.md#version-information
Therefore I am closing the issue.
Please let us know if there are still any problems or if an older version of Jobserver is needed.

**Comment by vimal3271** [comment]
@valan4ik , Could you please update 0.6.2 jar also ? We are using old version of spark(2.2.1)

**Comment by sgautrin** [comment]
@valan4ik Could you provide version 0.8.0 as well? This is still the version used by the [latest Datastax Enterprise (6.8)](https://docs.datastax.com/en/dse/6.8/dse-admin/datastax_enterprise/releaseNotes/RNdse.html#RNdse683__683comp), so any project that needs to run on that has to declare version 0.8.0 as provided and thus needs to be able to get it.

Thanks.

**Comment by valan4ik** [comment]
Hi @sgautrin ,

I can't release `0.8.0` once again, but I published a new `0.8.1` release. Hope that will work for you.

Best,
Valentina

**Comment by sgautrin** [comment]
This should @valan4ik thanks. I'll confirm when I get back from my days off work, but I don't see why it would not work for us at least, just updating to 0.8.1 in our code base.
